concepts/index.md

```diff
@@ -3,47 +3,56 @@ layout: single
+It provides two core sets of features: *cluster management* and *deployment management*. Here is an overview of these features:
 
-Server Groups mandatorily take on the naming convention *application-stack-detail-version*, where *stack* and *detail* are optional and *version* is system-assigned (incrementing version number). Spinnaker has no particular opinion on how the *stack* and *detail* fields are used, however it's common to use *stack* to represent a particular environment for a given microservice (e.g. dev, test, prod) and to use *detail* as an additional descriptor (e.g. feature branch, "experimental"). Finally, *version* indicates the specific Server Group, and individual instances of a Server Group are given a hash suffix in their names (e.g. "c9nzt") to uniquely identify each.
+## Cluster Management
 
-## Clusters
+You use Spinnaker's cluster management features to view and manage your resources in the cloud.
 
-Clusters are logical groupings of Server Groups, by *application-stack-detail*. Clusters are used in situations where multiple Server Groups for a given microservice need to be coordinated in a single context, for example, during a red/black deployment.
+![](clusters.png)
 
-## Applications
+### Server Group
 
-Applications are a logical grouping of infrastructure components - Server Groups and Clusters, Load Balancers, Security Groups and Pipelines. All components under a given Application share infrastructure credentials and top-level permissions. There are a few areas where cross-Application activities occur (e.g. a Stage that starts a Pipeline in another application), however in most areas, references are relegated to within the Application.
+The base resource, the *Server Group*, identifies the machine instance profile on which to execute images along with the number of instances. This resource is associated with a Load Balancer and a Security Group. A Server Group also has basic configuration settings, such as user account information and the region/zone in which images are deployed. When deployed, a Server Group is a collection of virtual machines running software.
 
-An Application typically corresponds to a particular microservice.
+### Cluster
 
-## Projects
+You can define *Clusters*, which are logical groupings of Server Groups in Spinnaker.
 
-Projects are arbitrary collections of infrastructure resources and pipelines. This is a convenience feature which allows you to create a dashboard containing Applications, Clusters and Pipelines by wildcard.
+### Load Balancer
 
-<!-- ## Infrastructure
+A *Load Balancer* is associated with an ingress protocol and port range. It balances traffic among instances in its Server Group. Optionally, you can enable health checks for a load balancer, with flexiblity to define health criteria and specify the health check endpoint.
```

Comment:
> in its Server Group

Multiple server groups can be added to a load balancer, so this should be "server groups"